Components/Installer/PaymentMethods/PaymentMethodsInstaller.php

```diff
@@ -574,27 +570,60 @@ private function setPaymentActive(Payment $payment, $isActive)
     /**
      * Returns a collection of active payment methods from the Mollie API.
      *
-     * @return null|BaseCollection|MethodCollection
+     * @return array
      */
     private function getActivePaymentMethodsFromMollie()
```

Comment:
there is still a api client as private member in this class.
i think we need to get rid off it right?
also, it might be good to use the api factory instead, then we have the factory as private member and always the need of thinking about what api client we need

Reply:
I already use the API factory. I create the factory within the `getActivePaymentMethodsFromMollie` method which is the only place it is used. So no need to pass that in the constructor. I'll remove the api client.

Reply:
i, alright,
the constructor was more like a nice to have to move it to the dependency, but its not mandatory
only removing the wrong api client :)
helemaal bedankt